### ISSUE TITLE
Apple Warranty Logo Not Showing UP

### ISSUE THREAD

**Issue by LSNIGHTSHOT** [grokability/snipe-it#12207]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When I enter my Apple products the Apple warranty logo that usually shows up in the Warranty section is not there.

### Reproduction steps

1. Click on Apple Asset logo is not there.
2.
3.
...

### Expected behavior

The logo to be there.

### Snipe-IT Version

6.0.13

### Operating System

Windows Server 2022

### Web Server

IIS

### PHP Version

8.1.13

### Operating System

Windows

### Browser

Edge

### Version

107.0.1418.62

**Comment by snipe** [comment]
I kinda need a little more info here. Was your browser recently updated? Was Snipe-IT recently updated? (We haven't made any changes to the logo or to that view in a few months, so...)

Some screenshots might be helpful.

**Comment by LSNIGHTSHOT** [comment]
Sorry it was very vague.

I have two instances on the same Snipe version and PHP version. Same browser version as well. Both have been updated for a week. One has it the other doesn't. Here is a screenshot of what I'm talking about. 

I did upgrade to version to 108.0.1462.41 and it still didn't fix it. 

![image](https://user-images.githubusercontent.com/73556517/206287549-efec9018-16fe-48e7-afda-94e9d572c914.png)

**Comment by snipe** [comment]
We haven't made any changes to that blade, so I'm not sure why you'd be seeing that tbh.

https://github.com/snipe/snipe-it/blob/97731cd7c812a16a8ccaab5b4ca33d6aed59c3b7/resources/views/hardware/view.blade.php#L587-L591

**Comment by snipe** [comment]
(Just caught a typo in that screenreader text - oops)

**Comment by LSNIGHTSHOT** [comment]
Is it necessary to have AppleCare for it to work? Is that the problem?

**Comment by snipe** [comment]
No, it does a simple check for "Apple" as the manufacturer name - it doesn't try to reach into Apple's API
